payload-schemas/schemas/common/github.schema.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
I think a better name for this would be `github-org` (or similar), since `GitHub` by itself is pretty ambogious and we could have similar common interfaces in the future.